src/test/resources/features/convertImage/image.feature

```diff
@@ -0,0 +1,283 @@
+      | extractThumbnail |  on                               |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "common/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200 |
+
+  @functional
+  Scenario: Verify that the image file is converted to grayscale
+    When I send a POST request to "/convertImage" with the following form data
+      | file             |  @"template/image/image.jpg"      |
+      | md5              |  1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .png                             |
+      | name             |  test                             |
+      | height           |  10                               |
+      | width            |  10                               |
+      | gray             |  on                               |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "common/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200 |
+
+  @functional
+  Scenario: Verify that the image file is resized
+    When I send a POST request to "/convertImage" with all the following form data
+      | file             |  @"template/image/image.jpg"      |
+      | md5              |  1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .png                             |
+      | name             |  test                             |
+      | changeSize       |  on                               |
+      | position         |  2,5                              |
+      | height           |  10                               |
+      | width            |  10                               |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "image/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200 |
+
+  @functional
+  Scenario: Verify that an image file can extract its metadata in json file
+    When I send a POST request to "/convertImage" with all the following form data
+      | file             |  @"template/image/image.jpg"      |
+      | md5              |  1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .png                             |
+      | name             |  test                             |
+      | extractMetadata  |  on                               |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "image/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200 |
+
+  @functional
+  Scenario: Verify that an image file can extract its thumbnail
```

Comment:
Please consider improving this scenario title to something like: `Verify that is possible to extract thumnail from an image file`